src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-gateway-group/nvmeof-gateway-group.component.ts

```diff
@@ -52,12 +57,17 @@ export class NvmeofGatewayGroupComponent implements OnInit {
   icons = Icons;
 
   iconSize = IconSize;
+  @ViewChild('deleteTpl', { static: true })
+  deleteTpl: TemplateRef<any>;
+  @Input() isNvme: boolean = false;
```

Comment:
why this ? whole file is nvme ?

or add a comment/explanation for adding or introducing new props.

Reply:
we can remove this now right? i don't see it being used anywhere.